test/jdk/java/util/Objects/BasicObjectsTest.java

```diff
@@ -134,6 +135,19 @@ private static int testToString2() {
         return errors;
     }
 
+    private static int testToDefaultString() {
+        int errors = 0;
+        try {
+            Objects.toDefaultString(null);
+            errors++;
+        } catch (NullPointerException npe) {
+            ; // Expected
+        }
+        Object o = new Object(){};
+        errors += (Objects.toDefaultString(o).equals(o.toString()))? 0 : 1;
+        return errors;
```

Comment:
Another potential test is on an instance of a class that overrides toString(), that would ensure that toDefaultString() doesn't invoke toString.